drivers/user_permission_report/app/controllers/user_permission_report/warehouse_reports/reports_controller.rb

```diff
@@ -0,0 +1,30 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module UserPermissionReport::WarehouseReports
+  class ReportsController < ApplicationController
+    include WarehouseReportAuthorization
+    include ArelHelper
+
+    before_action :set_limited, only: [:index]
```

Comment:
Since this report operates on users, not projects, and doesn't provide a project filter, I don't think this is useful.  I'm pretty sure there's not a helpful way to limit the users you can see at this time.

Reply:
✅